src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -127,18 +127,30 @@ function featureToPoint(feature, options) {
     const vertices = new Float32Array(ptsIn.length);
     const colors = new Uint8Array(ptsIn.length);
 
+    const batchIds = options.batchId ?  new Uint32Array(ptsIn.length / 3) : undefined;
+    let FeatureId = 0;
+
     coordinatesToVertices(ptsIn, normals, vertices, options.altitude);
 
     for (const geometry of feature.geometry) {
         const color = getProperty('color', options, randomColor, geometry.properties);
         const start = geometry.indices[0].offset;
         const count = geometry.indices[0].count;
         fillColorArray(colors, count, color, start);
+
+        if (batchIds) {
+            const id = options.batchId(geometry.properties, FeatureId);
+            for (let i = start; i < start + count; i++) {
```

Comment:
it will be better to do in the same loop of `fillColorArray`

Reply:
I think it is premature optimization : I want first to be able to support arbitrary numbers of optional custom attributes (see https://github.com/itownsResearch/sealevel/commit/a6ab40b97df824acf3260d3e095944e1f6c7433f).

Reply:
yes, we'll see later